serialization/physics.proto

```diff
@@ -116,13 +116,29 @@ message HierarchicalSystem {
 
 message KeplerianElements {
   required Frame frame = 1;
-  required double eccentricity = 2;
+  optional double eccentricity = 2;
```

Comment:
As discussed, optionals are fine. Comments would be redundant with those in the struct since this mirrors the struct, let's not replicate comments.